pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -813,6 +817,32 @@ cdef class _Timedelta(timedelta):
         else:
             return "D"
 
+    @property
+    def resolution_timedelta(self):
```

Comment:
The general idea was to have this new property return `Timedelta(nanosecond=1)` akin to how `datetime.timedelta.resolution` returns `timedelta(microseconds=1)`